### DISCUSSION TITLE
Few Suggestions for Admin Dashboard

### DISCUSSION THREAD

**Discussion by osseonews** [swellstores/community#21] [Ideas]
Here are few suggestions for your admin dashboard to make it more practical to use for most businesses, almost all of which still receive phone in orders:

1. **Address book:** On the customer page (admin/customers/ID): Have a list of all the customer addresses there. You list orders and other information, but the customers address book is not there. Most customers will have multiple shipping addresses and a customer rep needs to see that information. Plus they need to be able to add a new address to the address book, not just edit the default address. 

2.  **Pre-fill billing address for new card:** If you are on the customer list and you click to add a New Card, you are prompted to enter billing information. It says it's optional, but it's not. Please pre-fill this billing information for any New Card with the customers existing billing information. Right now, we have to retype the customer billing address manually for every new card and this is very annoying, wastes time, and prone to errors. Just pre-fill the default billing address on the New Card form.

3. **Allow selection of Address entry from the Address bok when creating an order.** When you create an order for a customer, the only address you can use is the default shipping address. But let's say a customer has multiple addresses and you need to choose one that is not the default? You then have to retype the address again, even if it is already in the customer's address book. When creating an order, instead of having to manually type in a new order, have a selection field, which allows you to select an address from the customer's list of addresses that will then pre-fill the shipping address.

Thanks.

**Comment by kyle-swell** [comment]
@osseonews Thanks for the suggestions. I agree with all of them! I will forward these suggestions to our product team. 😃

**Comment by is-hiku** [comment]
or make the Admin extensible so that we can customize it based on our needs.

**Reply by ericingram** [reply]
We are definitely doing that

**Comment by ericingram** [comment]
Good ideas!

**Comment by papertokyo** [comment]
+1 for all of these
